.github/actions/render/action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
     default: 'master'
   DITA_OT_VERSION:
     description: 'DITA-OT version used for rendering'
-    default: '4.0.1'
+    default: '4.1.2'
```

Comment:
@jelovirt Could we use a variable here?

Reply:
That's the default value for the variable. So in a way it's already a variable.